meecrowave-core/src/main/java/org/apache/meecrowave/Meecrowave.java

```diff
@@ -127,6 +137,8 @@
     private Runnable postTask;
     private boolean clearCatalinaSystemProperties;
 
+    private static final LogFacade LOGGER = new LogFacade(Meecrowave.class.getName());
```

Comment:
this one does not work since it will trigger an eager init whereas we want a lazy one (false positive pby)